lib/foodAjax.js

```diff
@@ -16,8 +16,24 @@ function postAjax(formattedFood) {
     data: formattedFood,
     dataType: "json",
     success: function mediaData(data){
-        console.log(data);
-        addNewFoodToTable(data);
+      addNewFoodToTable(data);
     }
   })
 }
+
+function editAjax(food) {
+  $.ajax({
+    method: 'PATCH',
+    url: 'http://localhost:3000/api/v1/foods/' + food.id,
+    data: {food: {name: food.name, calories: food.calories}}
+  }).done()
+  .catch(logErrors);
+}
+
+function deleteAjax(id) {
+  $.ajax({
+    method: 'DELETE',
+    url: "http://localhost:3000/api/v1/foods/" + id,
+  }).done()
+  .catch(logErrors)
+}
```

Comment:
I pulled both of these ajax calls out of their previous homes and put them in this ajax file